src/discretize.jl

```diff
@@ -81,14 +81,21 @@ Exact discretization algorithm for affine systems.
 
 ### Algorithm
 
-Consider a `NoisyAffineControlledContinuousSystem` with system dynamics
-``x' = Ax + Bu + c + Dw``.
-
-The exact discretization is calculated by solving the integral for
-``t = [t, t + ΔT]`` for a fixed input `u` and fixed noise realization `w`.
+This algorithm consists of integrating the continuous differential equation over
+a specified time interval to obtain an associated discrete differential equation.
```

Comment:
```suggestion
a specified time interval to obtain an associated recurrence relation.
```
While the old term exists (and also the term "difference relation"), I prefer [recurrence relation](https://en.wikipedia.org/wiki/Recurrence_relation).

Reply:
I have never heard `recurrence relation` in this context before.  But yeah, also makes sense. I am use to the term `difference equation` . Could this be an alternative?

Reply:
Yeah, I meant "difference *equation*". In the end, I am fine with everything, it was just a suggestion. I would get confused if I read "discrete differential". Are there other opinions?

Reply:
I can't think of any at the moment.

Reply:
*opinions*, not *options* :wink:
@blegat @mforets?

Reply:
In the context of this library, we call such systems discrete systems. I would use that name. Bu both the terms "Discrete differential equations" and "recurrence relation" are also correct imo.